api/src/main/java/org/openmrs/util/OpenmrsConstants.java

```diff
@@ -635,6 +634,11 @@ public static final Collection<String> AUTO_ROLES() {
 	 * Global property that stores the base url for the password reset.
 	 */
 	public static final String GP_PASSWORD_RESET_URL = "security.passwordResetUrl";
+
+	/**
+	 * Global property that stores the number of days for users to be deactivated.
+	 */
+	public static final String GP_NUMBER_OF_DAYS_TO_AUTO_RETIRE_USERS = "users.numberOfDaysToRetire";
```

Comment:
Can we also add this to CORE_GLOBAL_PROPERTIES?

Reply:
Done